server/src/event/model/create_taxonomy_link.rs

```diff
@@ -0,0 +1,73 @@
+use std::collections::HashMap;
+use std::convert::TryFrom;
+
+use crate::database::Executor;
+use serde::Serialize;
+
+use super::{AbstractEvent, Event, EventError, EventPayload, RawEventType};
+
+#[derive(Debug, Eq, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CreateTaxonomyLinkEvent {
+    entity_id: i32,
+    entity_revision_id: i32,
```

Comment:
```suggestion
    taxonomy_id: i32,
```

?

Reply:
I ask myself too. And there are a lot of mistakes in the file.
Corrected in [bc2b8c1](https://github.com/serlo/serlo.org-database-layer/pull/181/commits/bc2b8c14b8bd1212269e5c245a807cac0e9c08af)

Reply:
But I'm not sure about the following asymmetry:
In Create Entity Link Event (https://github.com/serlo/serlo.org-database-layer/blob/entity-create-patch/server/tests/entity.rs#L962-L967) the bearer of the `object_id` is the child.

But in the Create Taxonomy Link Event (https://github.com/serlo/serlo.org-database-layer/blob/entity-create-patch/server/tests/entity.rs#L902-L907) the `object_id` refers to the "parent".

I'm basing on the following result of the DB:
```sql
mysql> select event_log.uuid_id as 'uuid of taxonomy parent', event.name, event_parameter_uuid.uuid_id as 'uuid of entity child' f
rom event_log join event_parameter on log_id = event_log.id join event_parameter_uuid on event_parameter.id = event_parameter_id j
oin event on event.id = event_id  where event_log.id = 299;
+-------------------------+-------------------------+----------------------+
| uuid of taxonomy parent | name                    | uuid of entity child |
+-------------------------+-------------------------+----------------------+
|                       7 | taxonomy/term/associate |                 1495 |
+-------------------------+-------------------------+----------------------+

```

Reply:
clared